Cargo.toml

```diff
@@ -15,7 +15,8 @@ name = "rwcst"
 [workspace]
 members = [
         "dummy",
-        "actix_full"
+        "actix_full",
+	"reqwest"
```

Comment:
"reqwest" alone is not a good name. First because we will have more examples using it; and also because it matches the name of one of the crates being used, the ambiguity can make cargo confused at some points. I suggest using "actix_reqwest"